src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -37,9 +39,11 @@ public LoginSsoPageViewModel()
             _cryptoFunctionService = ServiceContainer.Resolve<ICryptoFunctionService>("cryptoFunctionService");
             _platformUtilsService = ServiceContainer.Resolve<IPlatformUtilsService>("platformUtilsService");
             _stateService = ServiceContainer.Resolve<IStateService>("stateService");
+            _logger = ServiceContainer.Resolve<ILogger>("logger");
+
 
             PageTitle = AppResources.Bitwarden;
-            LogInCommand = new Command(async () => await LogInAsync());
+            LogInCommand = new AsyncCommand(async () => await LogInAsync());
```

Comment:
you're missing here the flag for concurrent executions and you can spare the async await:
`LogInCommand = new AsyncCommand(LogInAsync, allowsMultipleExecutions: false); `